### PR TITLE
[4.13.100] Optimize initialization speed

### PR DESCRIPTION
**Pull request by yushih**
Please review commit https://github.com/Emurgo/yoroi-frontend/commit/c5d2601ca02ff80565a78dc95284c549c74351c0 individual if it helps.

 [ee7d093](https://github.com/Emurgo/yoroi-frontend/pull/2841/commits/ee7d093a2072ad7d731883b891978881b5d2f1a9) makes perceived loading speed faster by loading wallet data after the main UI starts to render.

  https://github.com/Emurgo/yoroi-frontend/pull/2841/commits/80f85766a1a019b28ec1dd789c914d76b1e590d8 loads SVGs on-demand (some of them are huge but rarely used). But because of it, all the SVG import statements need to be rewritten: c5d2601ca02ff80565a78dc95284c549c74351c0 is just mechanical changing of all import syntax for SVGs. Also, flow type of SVG imports must be fixed by bd83cd2e36726bc3637c8273629e42e37b4a9b2c.

Other commits should be straightforward by the commit messages.

### CODE REVIEW

File: packages/yoroi-extension/app/components/common/CheckboxLabel.js [hunk 1]
```diff
@@ -2,8 +2,8 @@
 import type { Node } from 'react';
 import { Checkbox, FormControlLabel, Typography } from '@mui/material';
 
-import OutlineIcon from '../../assets/images/forms/checkbox-outline.inline.svg';
-import CheckedIcon from '../../assets/images/forms/checkbox-checked.inline.svg';
+import { ReactComponent as OutlineIcon}  from '../../assets/images/forms/checkbox-outline.inline.svg';
+import { ReactComponent as CheckedIcon}  from '../../assets/images/forms/checkbox-checked.inline.svg';
```
Comment (neuodev) [inline packages/yoroi-extension/app/components/common/CheckboxLabel.js]: Is it still necessary to have the `.inline` extension?

File: packages/yoroi-extension/app/components/common/Autocomplete.js [hunk 2]
```diff
@@ -7,9 +7,9 @@ import { useCombobox, useMultipleSelection } from 'downshift';
 import { Input, Box, InputLabel, FormControl, FormHelperText, Chip, useTheme } from '@mui/material';
 import { styled } from '@mui/system';
 import { slice } from 'lodash';
-import SuccessIcon from '../../assets/images/forms/done.inline.svg';
-import ErrorIcon from '../../assets/images/forms/error.inline.svg';
-import CloseIcon from '../../assets/images/close-chip.inline.svg';
+import { ReactComponent as SuccessIcon}  from '../../assets/images/forms/done.inline.svg';
```
Comment (neuodev) [inline packages/yoroi-extension/app/components/common/Autocomplete.js]: Please add one extra space after the closing bracket ` }` and this applies for every import statment!

**Review by neuodev — DISMISSED** [review]

File: packages/yoroi-extension/app/components/common/CheckboxLabel.js (hunk 1 above)
Comment (yushih) [reply packages/yoroi-extension/app/components/common/CheckboxLabel.js]: I don't know why the extension was there but I guess it's no harm.

File: packages/yoroi-extension/app/components/common/Autocomplete.js (hunk 2 above)
Comment (yushih) [reply packages/yoroi-extension/app/components/common/Autocomplete.js]: Fixed.

File: packages/yoroi-extension/app/api/ada/paperWallet/paperWalletPdf.js [hunk 3]
```diff
@@ -39,7 +39,10 @@ export const generateAdaPaperPdf = async (
 
   const width = 595.28;
   const height = 841.98;
-  const doc = new Pdf({
+
+  const JsPdf = (await import('jspdf')).default;
```
Comment (zuzunker) [inline packages/yoroi-extension/app/api/ada/paperWallet/paperWalletPdf.js:43]: nice

**Review by zuzunker — DISMISSED** [review]
Thank you, @yushih !

**Review by zuzunker — CHANGES_REQUESTED** [review]
/check

(accidental request changes)

**Review by zuzunker — DISMISSED** [review]

**Review by zuzunker — COMMENTED** [review]
/check

File: packages/yoroi-extension/jest.config.js [hunk 4]
```diff
@@ -17,5 +17,6 @@ module.exports = {
     'jest-canvas-mock'
   ],
   // ignore manifest.test.js file, because it isn't a test
-  testPathIgnorePatterns: ['manifest.test.js']
+  testPathIgnorePatterns: ['manifest.test.js'],
+  modulePathIgnorePatterns: ['jormungandr'],
```
Comment (yushih) [inline packages/yoroi-extension/jest.config.js:21]: Oops, forgot about this.

**Review by zuzunker — DISMISSED** [review]

**Review by Nebyt — COMMENTED** [review]
/check

**Review by Nebyt — DISMISSED** [review]

**Review by Nebyt — COMMENTED** [review]
/check

**Review by Nebyt — DISMISSED** [review]

**Comment by zuzunker** [comment]
Release: https://github.com/Emurgo/yoroi-frontend/releases/tag/4.13.100

**Pull request merged**